### PR TITLE
Prs/update makefile

### PR DESCRIPTION
**Pull request by boekhorstb1**
updates to Makefile in relation to update to README.md in this PR here https://github.com/SovereignCloudStack/rookify/pull/72

### CODE REVIEW

**Comment by boekhorstb1** [comment]
note: currently branch is set up to merge into prs/enhance-ceph-pools

File: osism/get_configs_from_testbed.sh
Comment (boekhorstb1) [inline osism/get_configs_from_testbed.sh:1]: change filename: get_configs_from_osism_testbed.sh

File: scripts/osism/get_osism_configs_from_testbed.sh
Comment (NotTheEvilOne) [inline scripts/osism/get_osism_configs_from_testbed.sh:1]: Please remove "osism" from the file name as this duplicates the directory name and is referenced (without "osism") from `Makefile`.

File: Makefile [hunk 1]
```diff
@@ -75,15 +75,18 @@ build-container: ## Build container from Dockerfile, add e.g. ROOKIFY_VERSION=0.
 
 .PHONY: run-local-rookify
 run-local-rookify: ## Runs rookify in the local development environment (requires setup-venv)
-	source ./.venv/bin/activate && pip install -e . && rookify
+	if [ ! -f ./.venv/bin/rookify ]; then \
```
Comment (NotTheEvilOne) [inline Makefile:78]: Please use `Makefile` target dependencies to execute dependent rules.

```
.PHONY: run-local-rookify
run-local-rookify: build-local-rookify
```

**Review by NotTheEvilOne — CHANGES_REQUESTED** [review]

**Comment by boekhorstb1** [comment]
updated branch from main and reorganized consequently :)

File: scripts/osism/get_osism_configs_from_testbed.sh
Comment (NotTheEvilOne) [inline scripts/osism/get_osism_configs_from_testbed.sh:1]: I would rather like the second "osism" (in file name) to be removed.

Comment (boekhorstb1) [reply scripts/osism/get_osism_configs_from_testbed.sh:1]: allright

File: README.md
Comment (NotTheEvilOne) [inline README.md:1]: Adaptions to `README.md` are underway in #72. This file should not be changed here please :)

**Review by NotTheEvilOne — COMMENTED** [review]
LGTM so far, please consider reverting your change in `README.md` to prevent merge conflicts with #72

**Review by NotTheEvilOne — APPROVED** [review]

**Pull request merged**